xbmc/dialogs/GUIDialogSimpleMenu.cpp

```diff
@@ -90,12 +91,31 @@ bool CGUIDialogSimpleMenu::ShowPlaySelection(CFileItem& item, const std::string&
 
   CFileItemList items;
 
-  if (!GetDirectoryItems(directory, items, XFILE::CDirectory::CHints()))
+  if (item.GetVideoContentType() == VideoDbContentType::EPISODES)
   {
-    CLog::Log(LOGERROR,
-              "CGUIWindowVideoBase::ShowPlaySelection - Failed to get play directory for {}",
-              directory);
-    return true;
+    // See if can simplify titles
+    // Get episodes on disc
+    std::vector<CVideoInfoTag> episodes;
+    CVideoDatabase database;
+    database.Open();
+    database.GetEpisodesByFile(item.GetPath(), episodes);
+    database.Close();
+
+    const CURL url(directory);
+    auto br = XFILE::CBlurayDirectory();
+    br.GetEpisodeDirectory(url, item, items, episodes);
+  }
+
+  if (items.IsEmpty())
+  {
+    // Not episode or new episode search failed, so do it the old way
+    if (!GetDirectoryItems(directory, items, XFILE::CDirectory::CHints()))
+    {
+      CLog::Log(LOGERROR,
+                "CGUIWindowVideoBase::ShowPlaySelection - Failed to get play directory for {}",
```

Comment:
Class + method name wrong . To avoid logging wrong method names, use `CLog::LogF(...`.

Reply:
Fixed. That was actually legacy code.